config/example_test.go

```diff
@@ -34,10 +35,10 @@ func ExampleWithSharedConfigProfile() {
 }
 
 func ExampleWithCredentialsProvider() {
-	cfg, err := external.LoadDefaultAWSConfig(
+	cfg, err := config.LoadDefaultAWSConfig(
```

Comment:
Nit: Does LoadDefaultAWSConfig need to have AWS in it? What other type of Config can it load?

Reply:
Mainly had `AWS` in there to clarify the `AWSConfig` as the AWS shared config, shared credentials. Its kind of implied, but helped emphasis this.  But in theory this could support other.